### ISSUE TITLE
[UI] Theme colour should be consistent in setting page in dark mode

### ISSUE THREAD

**Issue by Yashsharma1911** [meshery/meshery#7092]
#### Current Behavior
Theme color should be consistent in setting page in dark mode as it is in light mode

https://user-images.githubusercontent.com/71271069/219146017-747e6aff-b655-4b9d-be4c-20b43a215590.mov

#### Desired Behavior

![Screenshot 2023-02-16 at 1 33 46 AM](https://user-images.githubusercontent.com/71271069/219146532-f74faf0e-40de-42fa-9bdc-cd8166c65b29.png)

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by acald-creator** [comment]
@Yashsharma1911 Is it just this page, or are there others that may need this as well?

**Comment by shruti1421** [comment]
@acald-creator, you are correct. I'm wondering the same thing. In addition, @Yashsharma1911 I'd want to work on this issue.

**Comment by Yashsharma1911** [comment]
Hey @shruti1421 thanks for raising your hand just to let you know that I am already working on it however there may be some other related issues that you could help with
